sm_td.h

```diff
@@ -38,7 +47,7 @@
 #define SMTD_GLOBAL_SIMULTANEOUS_PRESSES_DELAY_MS 0
 #endif
 
-#if SMTD_GLOBAL_SIMULTANEOUS_PRESSES_DELAY_MS > 0
+#ifdef SMTD_GLOBAL_SIMULTANEOUS_PRESSES_DELAY_MS > 0
```

Comment:
Shouldn't this be `#ifdef SMTD_GLOBAL_SIMULTANEOUS_PRESSES_DELAY_MS` without the `>` sign?

With it we get this compilation error:

```
sm_td.h:50:50: error: extra tokens at end of #ifdef directive [-Werror]
   50 | #ifdef SMTD_GLOBAL_SIMULTANEOUS_PRESSES_DELAY_MS > 0
      |                                                  ^
```